rate_limiter.rst

```diff
@@ -212,6 +212,47 @@ processes by reserving unused tokens.
             $limit->wait();
         } while (!$limit->isAccepted());
 
+Exposing the Rate Limiter Status
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using a rate limiter in APIs, it's common to include some standard HTTP
+headers in the response to expose the limit status (e.g. remaining tokens, when
+new tokens will be available, etc.)
+
+That's why the ``consume()`` object returns a :class:`Symfony\\Component\\RateLimiter\\RateLimit`
```

Comment:
The `reserve()` method returns a `Reservation` object that also has a `getRateLimit()` method.